src/composables/useGetMessages.ts

```diff
@@ -499,6 +509,7 @@ export function useGetMessagesProvider() {
 				requestId: token,
 			})
 			debugTimer.end(`${token} | long polling`, 'status 200')
+			tryChatRelay()
```

Comment:
This is not a guaranteed indicator that end of chat is reached, 304 response is.